Makefile.toml

```diff
@@ -120,7 +120,7 @@ category = "CI"
 script_runner = "@duckscript"
 script = '''
 exec rustup install 1.61 --profile minimal
-exec cargo +1.61 check-all-features
+exec cargo +1.61 make check-all-features
```

Comment:
sliightly worried that `cargo +foo make` won't work well all the time due to the toolchain bopping between cargo-make and cargo (we've had problems with this before, and the cargo team has said that cargo-make's use case is somewhat niche here)

Reply:
If I make the `check-all-features` job do `cargo --version` this gives me 1.61. Anyway, this is only a problem for the features job, most MSRV violations will be caught by the check job which doesn't recursively call `cargo-make`.